### PR TITLE
setup_engine: Ignore errors on downgrade

### PR DESCRIPTION
**Pull request by tinez**
Signed-off-by: Marcin Sobczyk <msobczyk@redhat.com>

### CODE REVIEW

**Comment by erav** [comment]
concise but difficult to understand why this syntax is used.

File: common/deploy-scripts/setup_engine.sh [hunk 1]
```diff
@@ -44,5 +44,5 @@ gpgcheck=0
 enabled=1
 EOF
   dnf downgrade -y --repo fast-datapath ovn-2021-central-21.12.0-46.el8fdp
```
Comment (michalskrivanek) [inline common/deploy-scripts/setup_engine.sh:46]: can you please use --repofrompath instead?

**Comment by michalskrivanek** [comment]
how about this?

File: common/deploy-scripts/setup_engine.sh [hunk 2]
```diff
@@ -34,15 +34,6 @@ cp /usr/share/doc/ovirt-engine/mibs/* /usr/share/snmp/mibs
...
-[fast-datapath]
-name=fast-datapath
-baseurl=http://rhsm-pulp.corp.redhat.com/content/dist/layered/rhel8/x86_64/fast-datapath/os
-gpgcheck=0
-enabled=1
-EOF
-  dnf downgrade -y --repo fast-datapath ovn-2021-central-21.12.0-46.el8fdp
-} || true
+# TODO: work around for https://bugzilla.redhat.com/2091565 - Downgrade OVN to 21.12.0-46 to match version on RHVH
+rpm -q rhvm && dnf downgrade -y dnf downgrade --repofrompath fdp,http://rhsm-pulp.corp.redhat.com/content/dist/layered/rhel8/x86_64/fast-datapath/os --repo fdp ovn-2021-central-21.12.0-46.el8fdp || true
```
Comment (erav) [inline common/deploy-scripts/setup_engine.sh]: double `dnf downgrade` ?

Comment (erav) [reply common/deploy-scripts/setup_engine.sh]: --repofrompath and then --repo again?

Comment (michalskrivanek) [reply common/deploy-scripts/setup_engine.sh]: yes, both are needed

**Pull request merged**